src/main/java/pl/sda/javapoz/security/SecurityConfig.java

```diff
@@ -24,50 +26,55 @@
     private UsersConnectionRepository usersConnectionRepository;
     private FacebookConnectionSignup facebookConnectionSignup;
     private UserService userService;
+    private MyUserDetailService myUserDetailService;
 
     @Autowired
-    public SecurityConfig(ConnectionFactoryLocator connectionFactoryLocator, UsersConnectionRepository usersConnectionRepository, FacebookConnectionSignup facebookConnectionSignup, UserService userService) {
+    public SecurityConfig(ConnectionFactoryLocator connectionFactoryLocator, UsersConnectionRepository usersConnectionRepository, FacebookConnectionSignup facebookConnectionSignup, UserService userService, MyUserDetailService myUserDetailService) {
         this.connectionFactoryLocator = connectionFactoryLocator;
         this.usersConnectionRepository = usersConnectionRepository;
         this.facebookConnectionSignup = facebookConnectionSignup;
         this.userService = userService;
+        this.myUserDetailService = myUserDetailService;
     }
 
     @Bean
     public CustomLogoutHandler customLogoutHandler() {
         return new CustomLogoutHandler();
     }
 
-
-    @Bean
-    public CustomUserDetailsService customUserDetailsService() {
-        return new CustomUserDetailsService();
-    }
-
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-
-
         http
                 .csrf().disable()
                 .authorizeRequests()
                 .antMatchers("/resources/**", "/static**").permitAll()
                 .antMatchers("/user").hasAnyRole("USER", "ADMIN")
                 .antMatchers("/linkForm").hasRole("ADMIN")
-                .antMatchers("/shop", "/beans","/").permitAll()
-                .antMatchers("/register","/news/**").permitAll()
-                .antMatchers("/contact","/product/**").permitAll()
+                .antMatchers("/shop", "/beans", "/").permitAll()
+                .antMatchers("/register", "/news/**").permitAll()
+                .antMatchers("/contact", "/product/**").permitAll()
                 .antMatchers("/login*", "/signin/**", "/signup/**").permitAll()
                 .anyRequest().authenticated()
                 .and()
-                .formLogin().loginPage("/login").successForwardUrl("/")
```

Comment:
To bym  zostawił

Reply:
z forwardem się wysypywało, a bez forwarda po zalogowaniu i tak wchodzi na główną stronę sklepu